package.json

```
@@ -120,6 +124,7 @@
 		"cssnano": "4.1.10",
 		"deep-freeze": "0.0.1",
 		"enzyme": "3.9.0",
+		"equivalent-key-map": "^0.2.0",
```

It shouldn't use ranges:

```suggestion
		"equivalent-key-map": "0.2.2",
```